recipes/seqan-library/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "seqan-library" %}
+{% set version = "2.2.0" %}
+{% set sha256 = "b5c036a3d2fc2fe5f2d57dcd4d7c523a6df146ab7b44bc789f42004b058d72fd" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.xz
+  url: https://github.com/seqan/seqan/releases/download/seqan-v{{ version }}/{{ name }}-{{ version }}.tar.xz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: cp -r include share $PREFIX  # [unix]
+
+requirements:
+  build:
+    - python >=3.3  # [win]
```

Comment:
Is Python required? Or is this being used to skip Python versions < 3.3? If so, you can use the skip tag - see [here](